common/src/main/java/com/wynntils/models/players/PartyModel.java

```diff
@@ -122,23 +123,21 @@ public void onWorldStateChange(WorldStateEvent event) {
     public void onChatReceived(ChatMessageReceivedEvent event) {
         if (event.getMessageType() != MessageType.FOREGROUND) return;
 
-        StyledText coded = event.getOriginalStyledText();
```

Comment:
I don't think this has any practical change, but it will open up for the possibility of you getting a message that has been modified by another listener.

Was there any specific reason to change this, or just that the code became shorter?

Reply:
We shouldn't be getting modified messages since this has EventPriority.HIGHEST. But using our StyledText makes the color codes more consistent, no random reset codes anywhere.

Reply:
Both event.getOriginalStyledText() and event.getStyledText() should return a StyledText, so there should be no difference in that regard..?

Reply:
isn't .getOriginalStyledText exactly what the server sends us?

Reply:
No, for that you need the originalComponent. The StyledText is always constructed by us, it is not a vanilla concept.

But this event is a bit confusing. We should probably give it some TLC.